ts/polyline.directive.ts

```diff
@@ -26,44 +26,266 @@ import { LayerProvider } from './layer.provider';
 import { lng2lat } from './lng2lat';
 import { MapComponent } from './map.component';
 
+/**
+ * Angular2 directive for Leaflet polylines.
+ *
+ * *You can use this directive in an Angular2 template after importing `YagaModule`.*
+ *
+ * How to use in a template:
+ * ```html
+ * <yaga-map>
+ *     <yaga-polyline
+ *         [(display)]="..."
+ *         [(stroke)]="..."
+ *         [(color)]="..."
+ *         [(weight)]="..."
+ *         [(opacity)]="..."
+ *         [(lineCap)]="..."
+ *         [(lineJoin)]="..."
+ *         [(dashArray)]="..."
+ *         [(dashOffset)]="..."
+ *         [(fill)]="..."
+ *         [(fillColor)]="..."
+ *         [(fillOpacity)]="..."
+ *         [(fillRule)]="..."
+ *         [(renderer)]="..."
+ *         [(className)]="..."
+ *         [(style)]="..."
+ *         [(latLngs)]="..."
+ *         [(bounds)]="..."
+ *         [(north)]="..."
+ *         [(east)]="..."
+ *         [(south)]="..."
+ *         [(west)]="..."
```

Comment:
@atd-schubert i don't think bounds, notrh, east, south and west make really sense here